cmd/manager/main.go

```diff
@@ -113,10 +114,17 @@ func main() {
 	}
 
 	log.Print("Setting up instance controller")
+	discoveryClient, err := utils.GetDiscoveryClient(mgr)
```

Comment:
Lots of line changes deal with the fact that we wire the `discoverClient` through the workflow engine 🤷‍♂